master/internal/experiment/bulk_action.go

```diff
@@ -277,11 +248,10 @@ func CancelExperiments(ctx context.Context, system *actor.System,
 		}
 	}
 
-	var refs []*actor.Ref
+	var refs []Experiment
```

Comment:
nit, here and others, `var refs []Experiment` -> `refs := make(map[int]Experiment)` and 264 becomes `for id, ref := range refs` and we can save the `ref.GetID()`?